css/style.css

```diff
@@ -43,20 +47,72 @@ body {
 }
 
 .content {
+    flex-grow: 1;
     background-color: #83A298;
 }
 
+.text {
+    background-color: #ffffff;
+    margin: 0px;
+    font-size: 10pt;
+    border: 1px solid gray;
+    padding: 1px 2px;
+    width: auto;
+    height: auto;
+}
 
-/* Style the info */
+.just-text {
+    text-align: justify;
+}
 
-.info {
+.flex-container {
+    display: flex;
+    flex-direction: column;
     background-color: #BCCDC7;
-    padding: 10px;
     margin: 0px 20px;
 }
 
+.info-left {
+    background-color: #BCCDC7;
+    margin: 0px 15px 0px 0px;
```

Comment:
Can you replace this with `margin-right`? Or are the zero margins needed in order to ensure this element is displayed correctly? If so, then you need to think about your cascading rules.